src/client/src/components/pages/courses/courseDetails/index.jsx

```diff
@@ -20,7 +20,7 @@ export default class index extends Component {
     // eslint-disable-next-line no-unused-vars
     const { courseName, category, startDate, endDate, desc } = this.state;
     const id = this.props.match.params.id;
-    const data = await axios(`/course/${id}`);
+    const data = await axios(`/api/v2/course/${id}`);
```

Comment:
Hey @mohannadhanafi, this isn't a required change, just a suggestion for the future, but one thing I've done in my own projects is to wrap all of my axios calls in my own small api function / file.  Then, you can put any common logic for all of your app's axios calls in that one spot.  You could then have, for example, `const apiPath = 'api/v2'` at the top of that file and append it to every route, and you'd just have to change that one place instead of multiple places when you, for example, changed to `api/v3`.

In the future, this api function is a good place to add an auth header to all of your axios calls.  Another thing I've done in the past is used this one function to redirect users to the sign-in page for any api call responds with `401 UNAUTHORIZED`.